polkadot/node/core/pvf/common/src/worker/security/mod.rs

```diff
@@ -27,143 +27,30 @@
 //! - Restrict networking by blocking socket creation and io_uring.
 //! - Remove env vars
 
-use crate::{worker::WorkerKind, LOG_TARGET};
+use crate::{worker::WorkerInfo, LOG_TARGET};
 
+#[cfg(target_os = "linux")]
+pub mod change_root;
 #[cfg(target_os = "linux")]
 pub mod landlock;
-
 #[cfg(all(target_os = "linux", target_arch = "x86_64"))]
```

Comment:
@alindima to confirm, if we're only blocking `socket` and `io_uring`, can we remove the restriction on `target_arch = "x86_64"`? (Context: we're dropping the seccomp whitelist idea, see https://github.com/paritytech/polkadot-sdk/pull/2461 for explanation.)

Reply:
Nvm, I dig up this comment. We're waiting on a CI job for aarch64: https://github.com/paritytech/polkadot-sdk/pull/2009#discussion_r1371695338